code/modules/organs/pain.dm

```diff
@@ -59,7 +59,7 @@
 			if(91 to 10000)
 				flash_pain()
 				msg = "<font size=3>OH GOD! Your [damaged_organ.name] is [burning ? "on fire" : "hurting terribly"]!</font>"
-		custom_pain(msg, maxdam, prob(10))
+		custom_pain(msg, maxdam, prob(clamp(maxdam/10,0,100)))
```

Comment:
This should be a lot lower, I think this is organ damage, so the max health of most limbs is 60. 30 is a bone break in most cases, so you should probably get the most pain messages around that amount of damage on a limb.

Reply:
Broken limbs should cause Shock damage too, which should trigger other pain messages as well (ideally). 

However, could increase this probability.

Do note, this works on top of existing pain messages. Like - you are guaranteed 1 pain message per 90 seconds, plus a 3% chance EVERY LIFE TICK (so like every second or so) to ignore that cooldown.

Even at 3% chance, considering how often this is checked? I think it should triggered frequently enough.

Reply:
Its called every 2 seconds to be specific. Also its not 3% chance to ignore cooldown, its 3% chance to even trigger at all IF cooldown isnt currently in place. So its 2 minutes of cooldown and then a bunch more time with however it takes to trigger the message on top of that.

Reply:
You misunderstand.

`if(force || (message != last_pain_message) || (world.time >= next_pain_time))`

IF ANY of the following is true:
- We roll the 3% probability
- Cooldown expired
- It's NOT the last limb that was injured

This is explicitly a "ignore the cooldown."